include/zephyr/dt-bindings/espi/npcx_espi.h

```diff
@@ -6,6 +6,20 @@
 #ifndef ZEPHYR_INCLUDE_DT_BINDINGS_ESPI_NPCX_ESPI_H_
 #define ZEPHYR_INCLUDE_DT_BINDINGS_ESPI_NPCX_ESPI_H_
 
+/*
+ * Encode virtual wire information into a 16-bit unsigned.
+ * index  = bits[7:0], Replacement index number
+ * group = bits[11:8], Group number for VWEVMS or VWEVSM
+ * dir = bits[13:12], Direction for controller to target or target to controller
+ */
+#define ESPI_NPCX_VW_EX_VAL(dir, group, index)                                                     \
+	(((dir & 0x1) << 12) + ((group & 0xf) << 8) + ((index & 0xff)))
```

Comment:
last parenthesis is redundant? `(index & 0xff)`

Reply:
Done. Have removed the redundant.